README.md

```diff
@@ -92,7 +92,9 @@ render(<App />, document.getElementById('root'));
 
 ### For development
 
-1. At the project root, run `yarn install`
+1. Fork this repository using by clicking _Fork_ button at the top
+1. Clone your own version on repository by using `git clone https://github.com/<username>/reactfire.git` (replace the repo url with your own version's url)
+1. At the project root, install all modules by running `yarn install` or `npm install`
```

Comment:
Please remove `npm install`. This project uses a [yarn workspace](https://yarnpkg.com/lang/en/docs/workspaces/) so `npm install` likely won't work.